integration/embedded_derp_test.go

```diff
@@ -185,6 +188,34 @@ func derpServerScenario(
 
 	t.Logf("Run2: %d successful pings out of %d", success, len(allClients)*len(allHostnames))
 
+	hsServer, err := scenario.Headscale()
+	assertNoErrGetHeadscale(t, err)
+
+	derpRegion := tailcfg.DERPRegion{
+		RegionCode: "test-derpverify",
+		RegionName: "TestDerpVerify",
+		Nodes: []*tailcfg.DERPNode{
+			{
+				Name:             "TestDerpVerify",
+				RegionID:         900,
+				HostName:         hsServer.GetHostname(),
+				STUNPort:         3478,
+				STUNOnly:         false,
+				DERPPort:         443,
+				InsecureForTests: true,
+			},
+		},
+	}
+
+	fakeKey := key.NewNode()
+	DERPVerify(t, fakeKey, derpRegion, false)
```

Comment:
This is fine, but I think it should not run in the common scenario ,and only in a specific test for this DERP setup.